react/src/components/Chat/ChatMessageContainer.tsx

```diff
@@ -0,0 +1,55 @@
+import Flex, { FlexProps } from '../Flex';
+import { Avatar, theme } from 'antd';
+import React, { memo } from 'react';
+
+export type ChatMessagePlacement = {
+  top?: boolean;
+  left?: boolean;
+  right?: boolean;
+  bottom?: boolean;
+};
+
+export interface ChatMessageContainerProps extends FlexProps {
+  placement?: ChatMessagePlacement;
+  containerStyle?: React.CSSProperties;
+  avatar?: React.ReactNode;
+  children?: React.ReactNode;
+}
+
+export const ChatMessageContainer: React.FC<ChatMessageContainerProps> = memo(
```

Comment:
Add container to wrap a chat message. The component only focus to manage only layout of chat messages

Reply:
to reduce calculation for placement and style